2014/10/20 01:29:12 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'train.user_login' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `user_login`.
in C:\AppServ\www\train2\index.php (13)
2014/10/20 02:07:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "s".' in C:\AppServ\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('s')
#1 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\AppServ\www\train2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/train2/index.php/s
---
2014/10/20 02:07:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "scrach".' in C:\AppServ\www\yii\framework\web\CController.php:483
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CController.php(270): CController->missingAction('scrach')
#1 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('scrach')
#2 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/scrac...')
#3 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\AppServ\www\train2\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/train2/index.php/userlogin/scrach
---
2014/10/20 02:08:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "_search".' in C:\AppServ\www\yii\framework\web\CController.php:483
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CController.php(270): CController->missingAction('_search')
#1 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('_search')
#2 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/_sear...')
#3 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\AppServ\www\train2\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/train2/index.php/userlogin/_search
---
2014/10/20 03:00:35 [error] [php] mail() [<a href='function.mail'>function.mail</a>]: Failed to connect to mailserver at &quot;localhost&quot; port 25, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set() (C:\AppServ\www\train2\protected\controllers\SiteController.php:67)
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#1 C:\AppServ\www\yii\framework\web\CController.php(286): SiteController->runAction()
#2 C:\AppServ\www\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#3 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): SiteController->run()
#4 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#5 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 C:\AppServ\www\train2\index.php(13): CWebApplication->run()
REQUEST_URI=/train2/index.php/site/contact
in C:\AppServ\www\train2\protected\controllers\SiteController.php (67)
in C:\AppServ\www\train2\index.php (13)
2014/10/20 03:05:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "user/index".' in C:\AppServ\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('user/index')
#1 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\AppServ\www\train2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/train2/index.php/user/index
HTTP_REFERER=http://localhost/train2/
---
2014/10/20 03:05:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "user".' in C:\AppServ\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#1 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\AppServ\www\train2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/train2/index.php/user
---
2014/10/20 03:05:21 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "user/index".' in C:\AppServ\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('user/index')
#1 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\AppServ\www\train2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/train2/index.php/user/index
HTTP_REFERER=http://localhost/train2/index.php/
---
2014/10/20 03:05:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "user/index".' in C:\AppServ\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('user/index')
#1 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\AppServ\www\train2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/train2/index.php/user/index
HTTP_REFERER=http://localhost/train2/index.php/userlogin/create
---
2014/10/20 03:09:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "station/index".' in C:\AppServ\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('station/index')
#1 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\AppServ\www\train2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/train2/index.php/station/index
HTTP_REFERER=http://localhost/train2/index.php/userlogin/create
---
2014/10/20 03:09:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "line/index".' in C:\AppServ\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('line/index')
#1 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\AppServ\www\train2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/train2/index.php/line/index
HTTP_REFERER=http://localhost/train2/index.php/station/index
---
2014/10/20 03:09:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "station/index".' in C:\AppServ\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('station/index')
#1 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\AppServ\www\train2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/train2/index.php/station/index
HTTP_REFERER=http://localhost/train2/index.php/line/index
---
2014/10/20 03:09:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "vehicle/index".' in C:\AppServ\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('vehicle/index')
#1 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\AppServ\www\train2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/train2/index.php/vehicle/index
HTTP_REFERER=http://localhost/train2/index.php/station/index
---
2014/10/20 03:09:06 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "user/index".' in C:\AppServ\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('user/index')
#1 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\AppServ\www\train2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/train2/index.php/user/index
HTTP_REFERER=http://localhost/train2/index.php/vehicle/index
---
2014/10/20 03:10:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "user/index".' in C:\AppServ\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('user/index')
#1 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\AppServ\www\train2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/train2/index.php/user/index
HTTP_REFERER=http://localhost/train2/index.php/user/index
---
2014/10/20 03:11:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "line/index".' in C:\AppServ\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('line/index')
#1 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\AppServ\www\train2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/train2/index.php/line/index
HTTP_REFERER=http://localhost/train2/index.php
---
2014/10/20 03:32:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "station/index".' in C:\AppServ\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('station/index')
#1 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\AppServ\www\train2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/train2/index.php/station/index
HTTP_REFERER=http://localhost/train2/index.php/line/index
---
2014/10/20 03:32:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "station/index".' in C:\AppServ\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('station/index')
#1 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\AppServ\www\train2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/train2/index.php/station/index
HTTP_REFERER=http://localhost/train2/index.php/station/index
---
2014/10/20 03:32:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "vehicle/index".' in C:\AppServ\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('vehicle/index')
#1 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\AppServ\www\train2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/train2/index.php/vehicle/index
HTTP_REFERER=http://localhost/train2/index.php/station/index
---
2014/10/20 03:32:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "station/index".' in C:\AppServ\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('station/index')
#1 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\AppServ\www\train2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/train2/index.php/station/index
HTTP_REFERER=http://localhost/train2/index.php/line/index
---
2014/10/20 03:32:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "vehicle/index".' in C:\AppServ\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('vehicle/index')
#1 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\AppServ\www\train2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/train2/index.php/vehicle/index
HTTP_REFERER=http://localhost/train2/index.php/station/index
---
2014/10/20 03:32:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "vehicle/index".' in C:\AppServ\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('vehicle/index')
#1 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\AppServ\www\train2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/train2/index.php/vehicle/index
HTTP_REFERER=http://localhost/train2/index.php/vehicle/index
---
2014/10/20 03:32:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "driver/index".' in C:\AppServ\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('driver/index')
#1 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\AppServ\www\train2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/train2/index.php/driver/index
HTTP_REFERER=http://localhost/train2/index.php/site/login
---
2014/10/20 03:32:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "xml/index".' in C:\AppServ\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('xml/index')
#1 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\AppServ\www\train2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/train2/index.php/xml/index
HTTP_REFERER=http://localhost/train2/index.php/driver/index
---
2014/10/20 03:32:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "xml/index".' in C:\AppServ\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('xml/index')
#1 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\AppServ\www\train2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/train2/index.php/xml/index
HTTP_REFERER=http://localhost/train2/index.php/xml/index
---
2014/10/20 03:32:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "driver/index".' in C:\AppServ\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('driver/index')
#1 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\AppServ\www\train2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/train2/index.php/driver/index
HTTP_REFERER=http://localhost/train2/index.php/xml/index
---
2014/10/20 03:32:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "xml/index".' in C:\AppServ\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('xml/index')
#1 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\AppServ\www\train2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/train2/index.php/xml/index
HTTP_REFERER=http://localhost/train2/index.php/driver/index
---
2014/10/20 03:43:38 [error] [exception.CException] exception 'CException' with message 'Property "Userlogin.id_user" is not defined.' in C:\AppServ\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\AppServ\www\yii\framework\db\ar\CActiveRecord.php(143): CComponent->__get('id_user')
#1 C:\AppServ\www\train2\protected\components\UserIdentity.php(26): CActiveRecord->__get('id_user')
#2 C:\AppServ\www\train2\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#3 C:\AppServ\www\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#4 C:\AppServ\www\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\AppServ\www\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#6 C:\AppServ\www\train2\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\AppServ\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#14 C:\AppServ\www\train2\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/train2/index.php/site/login
HTTP_REFERER=http://localhost/train2/index.php/site/login
---
2014/10/20 03:44:35 [error] [exception.CException] exception 'CException' with message 'Property "Userlogin.id_user" is not defined.' in C:\AppServ\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\AppServ\www\yii\framework\db\ar\CActiveRecord.php(143): CComponent->__get('id_user')
#1 C:\AppServ\www\train2\protected\components\UserIdentity.php(29): CActiveRecord->__get('id_user')
#2 C:\AppServ\www\train2\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#3 C:\AppServ\www\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#4 C:\AppServ\www\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\AppServ\www\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#6 C:\AppServ\www\train2\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\AppServ\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#14 C:\AppServ\www\train2\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/train2/index.php/site/login
HTTP_REFERER=http://localhost/train2/index.php/site/login
---
2014/10/21 00:27:18 [error] [exception.CException] exception 'CException' with message 'Property "Userlogin.id" is not defined.' in C:\AppServ\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\AppServ\www\yii\framework\db\ar\CActiveRecord.php(143): CComponent->__get('id')
#1 C:\AppServ\www\train2\protected\views\userlogin\_view.php(9): CActiveRecord->__get('id')
#2 C:\AppServ\www\yii\framework\web\CBaseController.php(130): require('C:\AppServ\www\...')
#3 C:\AppServ\www\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\AppServ\www\...', Array, false)
#4 C:\AppServ\www\yii\framework\zii\widgets\CListView.php(281): CBaseController->renderFile('C:\AppServ\www\...', Array)
#5 C:\AppServ\www\yii\framework\zii\widgets\CBaseListView.php(160): CListView->renderItems()
#6 [internal function]: CBaseListView->renderSection(Array)
#7 C:\AppServ\www\yii\framework\zii\widgets\CBaseListView.php(143): preg_replace_callback('/{(\w+)}/', Array, '{summary}?{sort...')
#8 C:\AppServ\www\yii\framework\zii\widgets\CBaseListView.php(128): CBaseListView->renderContent()
#9 C:\AppServ\www\yii\framework\web\CBaseController.php(173): CBaseListView->run()
#10 C:\AppServ\www\train2\protected\views\userlogin\index.php(20): CBaseController->widget('zii.widgets.CLi...', Array)
#11 C:\AppServ\www\yii\framework\web\CBaseController.php(126): require('C:\AppServ\www\...')
#12 C:\AppServ\www\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\AppServ\www\...', Array, true)
#13 C:\AppServ\www\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\AppServ\www\...', Array, true)
#14 C:\AppServ\www\yii\framework\web\CController.php(782): CController->renderPartial('index', Array, true)
#15 C:\AppServ\www\train2\protected\controllers\UserloginController.php(128): CController->render('index', Array)
#16 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): UserloginController->actionIndex()
#17 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#18 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#19 C:\AppServ\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#20 C:\AppServ\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#21 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#22 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#23 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#24 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#25 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#26 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/index')
#27 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#28 C:\AppServ\www\train2\index.php(13): CApplication->run()
#29 {main}
REQUEST_URI=/train2/index.php/userlogin/index
HTTP_REFERER=http://localhost/train2/index.php/
---
2014/10/21 00:29:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "userlogin/index".' in C:\AppServ\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/index')
#1 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\AppServ\www\train2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/train2/index.php/userlogin/index
HTTP_REFERER=http://localhost/train2/index.php/
---
2014/10/21 00:29:44 [error] [exception.CException] exception 'CException' with message 'Userlogin and its behaviors do not have a method or closure named "validatePassword".' in C:\AppServ\www\yii\framework\base\CComponent.php:265
Stack trace:
#0 C:\AppServ\www\yii\framework\db\ar\CActiveRecord.php(225): CComponent->__call('validatePasswor...', Array)
#1 [internal function]: CActiveRecord->__call('validatePasswor...', Array)
#2 C:\AppServ\www\train2\protected\components\UserIdentity.php(25): Userlogin->validatePassword('12345')
#3 C:\AppServ\www\train2\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#4 C:\AppServ\www\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#5 C:\AppServ\www\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#6 C:\AppServ\www\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#7 C:\AppServ\www\train2\protected\controllers\SiteController.php(94): CModel->validate()
#8 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#9 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\AppServ\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('login')
#13 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#14 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\AppServ\www\train2\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/train2/index.php/site/login
HTTP_REFERER=http://localhost/train2/index.php/site/login
---
2014/10/21 00:44:08 [error] [exception.CException] exception 'CException' with message 'Property "UserIdentity._id" is not defined.' in C:\AppServ\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\AppServ\www\train2\protected\components\UserIdentity.php(40): CComponent->__get('_id')
#1 C:\AppServ\www\yii\framework\web\auth\CWebUser.php(223): UserIdentity->getId()
#2 C:\AppServ\www\train2\protected\models\LoginForm.php(71): CWebUser->login(Object(UserIdentity), 0)
#3 C:\AppServ\www\train2\protected\controllers\SiteController.php(94): LoginForm->login()
#4 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#5 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 C:\AppServ\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('login')
#9 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#10 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 C:\AppServ\www\train2\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/train2/index.php/site/login
HTTP_REFERER=http://localhost/train2/index.php/site/login
---
2014/10/21 00:44:52 [error] [exception.CException] exception 'CException' with message 'Property "UserIdentity._id" is not defined.' in C:\AppServ\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\AppServ\www\train2\protected\components\UserIdentity.php(40): CComponent->__get('_id')
#1 C:\AppServ\www\yii\framework\web\auth\CWebUser.php(223): UserIdentity->getId()
#2 C:\AppServ\www\train2\protected\models\LoginForm.php(71): CWebUser->login(Object(UserIdentity), 0)
#3 C:\AppServ\www\train2\protected\controllers\SiteController.php(94): LoginForm->login()
#4 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#5 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 C:\AppServ\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('login')
#9 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#10 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 C:\AppServ\www\train2\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/train2/index.php/site/login
HTTP_REFERER=http://localhost/train2/index.php/site/login
---
2014/10/21 00:45:28 [error] [exception.CException] exception 'CException' with message 'Property "UserIdentity._id" is not defined.' in C:\AppServ\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\AppServ\www\train2\protected\components\UserIdentity.php(40): CComponent->__get('_id')
#1 C:\AppServ\www\yii\framework\web\auth\CWebUser.php(223): UserIdentity->getId()
#2 C:\AppServ\www\train2\protected\models\LoginForm.php(71): CWebUser->login(Object(UserIdentity), 0)
#3 C:\AppServ\www\train2\protected\controllers\SiteController.php(94): LoginForm->login()
#4 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#5 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 C:\AppServ\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('login')
#9 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#10 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 C:\AppServ\www\train2\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/train2/index.php/site/login
HTTP_REFERER=http://localhost/train2/index.php/site/login
---
2014/10/21 00:46:21 [error] [exception.CException] exception 'CException' with message 'Property "Userlogin.user_id" is not defined.' in C:\AppServ\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\AppServ\www\yii\framework\db\ar\CActiveRecord.php(143): CComponent->__get('user_id')
#1 C:\AppServ\www\train2\protected\views\userlogin\_view.php(9): CActiveRecord->__get('user_id')
#2 C:\AppServ\www\yii\framework\web\CBaseController.php(130): require('C:\AppServ\www\...')
#3 C:\AppServ\www\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\AppServ\www\...', Array, false)
#4 C:\AppServ\www\yii\framework\zii\widgets\CListView.php(281): CBaseController->renderFile('C:\AppServ\www\...', Array)
#5 C:\AppServ\www\yii\framework\zii\widgets\CBaseListView.php(160): CListView->renderItems()
#6 [internal function]: CBaseListView->renderSection(Array)
#7 C:\AppServ\www\yii\framework\zii\widgets\CBaseListView.php(143): preg_replace_callback('/{(\w+)}/', Array, '{summary}?{sort...')
#8 C:\AppServ\www\yii\framework\zii\widgets\CBaseListView.php(128): CBaseListView->renderContent()
#9 C:\AppServ\www\yii\framework\web\CBaseController.php(173): CBaseListView->run()
#10 C:\AppServ\www\train2\protected\views\userlogin\index.php(20): CBaseController->widget('zii.widgets.CLi...', Array)
#11 C:\AppServ\www\yii\framework\web\CBaseController.php(126): require('C:\AppServ\www\...')
#12 C:\AppServ\www\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\AppServ\www\...', Array, true)
#13 C:\AppServ\www\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\AppServ\www\...', Array, true)
#14 C:\AppServ\www\yii\framework\web\CController.php(782): CController->renderPartial('index', Array, true)
#15 C:\AppServ\www\train2\protected\controllers\UserloginController.php(128): CController->render('index', Array)
#16 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): UserloginController->actionIndex()
#17 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#18 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#19 C:\AppServ\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#20 C:\AppServ\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#21 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#22 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#23 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#24 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#25 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#26 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/index')
#27 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#28 C:\AppServ\www\train2\index.php(13): CApplication->run()
#29 {main}
REQUEST_URI=/train2/index.php/userlogin/index
HTTP_REFERER=http://localhost/train2/index.php/site/login
---
2014/10/21 00:46:59 [error] [exception.CException] exception 'CException' with message 'Property "Userlogin.user_id" is not defined.' in C:\AppServ\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\AppServ\www\yii\framework\db\ar\CActiveRecord.php(143): CComponent->__get('user_id')
#1 C:\AppServ\www\train2\protected\views\userlogin\_view.php(9): CActiveRecord->__get('user_id')
#2 C:\AppServ\www\yii\framework\web\CBaseController.php(130): require('C:\AppServ\www\...')
#3 C:\AppServ\www\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\AppServ\www\...', Array, false)
#4 C:\AppServ\www\yii\framework\zii\widgets\CListView.php(281): CBaseController->renderFile('C:\AppServ\www\...', Array)
#5 C:\AppServ\www\yii\framework\zii\widgets\CBaseListView.php(160): CListView->renderItems()
#6 [internal function]: CBaseListView->renderSection(Array)
#7 C:\AppServ\www\yii\framework\zii\widgets\CBaseListView.php(143): preg_replace_callback('/{(\w+)}/', Array, '{summary}?{sort...')
#8 C:\AppServ\www\yii\framework\zii\widgets\CBaseListView.php(128): CBaseListView->renderContent()
#9 C:\AppServ\www\yii\framework\web\CBaseController.php(173): CBaseListView->run()
#10 C:\AppServ\www\train2\protected\views\userlogin\index.php(20): CBaseController->widget('zii.widgets.CLi...', Array)
#11 C:\AppServ\www\yii\framework\web\CBaseController.php(126): require('C:\AppServ\www\...')
#12 C:\AppServ\www\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\AppServ\www\...', Array, true)
#13 C:\AppServ\www\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\AppServ\www\...', Array, true)
#14 C:\AppServ\www\yii\framework\web\CController.php(782): CController->renderPartial('index', Array, true)
#15 C:\AppServ\www\train2\protected\controllers\UserloginController.php(128): CController->render('index', Array)
#16 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): UserloginController->actionIndex()
#17 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#18 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#19 C:\AppServ\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#20 C:\AppServ\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#21 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#22 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#23 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#24 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#25 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#26 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/index')
#27 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#28 C:\AppServ\www\train2\index.php(13): CApplication->run()
#29 {main}
REQUEST_URI=/train2/index.php/userlogin/index
HTTP_REFERER=http://localhost/train2/index.php/site/login
---
2014/10/21 00:47:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "gii".' in C:\AppServ\www\yii\framework\web\CController.php:483
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CController.php(270): CController->missingAction('gii')
#1 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('gii')
#2 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/gii')
#3 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\AppServ\www\train2\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/train2/index.php/site/gii
---
2014/10/21 00:50:16 [error] [exception.CException] exception 'CException' with message 'Property "UserIdentity._id" is not defined.' in C:\AppServ\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\AppServ\www\train2\protected\components\UserIdentity.php(40): CComponent->__get('_id')
#1 C:\AppServ\www\yii\framework\web\auth\CWebUser.php(223): UserIdentity->getId()
#2 C:\AppServ\www\train2\protected\models\LoginForm.php(71): CWebUser->login(Object(UserIdentity), 0)
#3 C:\AppServ\www\train2\protected\controllers\SiteController.php(94): LoginForm->login()
#4 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#5 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 C:\AppServ\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('login')
#9 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#10 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 C:\AppServ\www\train2\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/train2/index.php/site/login
HTTP_REFERER=http://localhost/train2/index.php/site/login
---
2014/10/21 00:51:51 [error] [exception.CException] exception 'CException' with message 'Property "UserIdentity._id" is not defined.' in C:\AppServ\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\AppServ\www\train2\protected\components\UserIdentity.php(39): CComponent->__get('_id')
#1 C:\AppServ\www\yii\framework\web\auth\CWebUser.php(223): UserIdentity->getId()
#2 C:\AppServ\www\train2\protected\models\LoginForm.php(71): CWebUser->login(Object(UserIdentity), 0)
#3 C:\AppServ\www\train2\protected\controllers\SiteController.php(94): LoginForm->login()
#4 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#5 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 C:\AppServ\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('login')
#9 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#10 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 C:\AppServ\www\train2\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/train2/index.php/site/login
HTTP_REFERER=http://localhost/train2/index.php/site/login
---
2014/10/21 01:09:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "train".' in C:\AppServ\www\yii\framework\web\CController.php:483
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CController.php(270): CController->missingAction('train')
#1 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('train')
#2 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/train')
#3 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\AppServ\www\train2\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/train2/index.php/userlogin/train
---
2014/10/21 01:11:27 [error] [exception.CException] exception 'CException' with message 'TrainController cannot find the requested view "admin".' in C:\AppServ\www\yii\framework\web\CController.php:878
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CController.php(782): CController->renderPartial('admin', Array, true)
#1 C:\AppServ\www\train2\protected\controllers\TrainController.php(143): CController->render('admin', Array)
#2 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): TrainController->actionAdmin()
#3 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#5 C:\AppServ\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 C:\AppServ\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#7 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#9 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#10 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#12 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('train/admin')
#13 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#14 C:\AppServ\www\train2\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/train2/index.php/train/admin
HTTP_REFERER=http://localhost/train2/index.php/train
---
2014/10/21 01:12:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "xml/index".' in C:\AppServ\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('xml/index')
#1 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\AppServ\www\train2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/train2/index.php/xml/index
HTTP_REFERER=http://localhost/train2/index.php/userlogin/index
---
2014/10/21 01:12:33 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "driver/index".' in C:\AppServ\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('driver/index')
#1 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\AppServ\www\train2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/train2/index.php/driver/index
HTTP_REFERER=http://localhost/train2/index.php/userlogin/index
---
2014/10/21 19:09:19 [error] [exception.CException] exception 'CException' with message 'Property "CWebUser.username" is not defined.' in C:\AppServ\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\AppServ\www\yii\framework\web\auth\CWebUser.php(140): CComponent->__get('username')
#1 C:\AppServ\www\train2\protected\views\layouts\main.php(38): CWebUser->__get('username')
#2 C:\AppServ\www\yii\framework\web\CBaseController.php(126): require('C:\AppServ\www\...')
#3 C:\AppServ\www\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\AppServ\www\...', Array, true)
#4 C:\AppServ\www\yii\framework\web\widgets\CContentDecorator.php(76): CBaseController->renderFile('C:\AppServ\www\...', Array, true)
#5 C:\AppServ\www\yii\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate('<div id="conten...')
#6 C:\AppServ\www\yii\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput('<div id="conten...')
#7 C:\AppServ\www\yii\framework\web\CBaseController.php(206): COutputProcessor->run()
#8 C:\AppServ\www\yii\framework\web\CBaseController.php(300): CBaseController->endWidget('CContentDecorat...')
#9 C:\AppServ\www\train2\protected\views\layouts\column1.php(6): CBaseController->endContent()
#10 C:\AppServ\www\yii\framework\web\CBaseController.php(126): require('C:\AppServ\www\...')
#11 C:\AppServ\www\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\AppServ\www\...', Array, true)
#12 C:\AppServ\www\yii\framework\web\CController.php(784): CBaseController->renderFile('C:\AppServ\www\...', Array, true)
#13 C:\AppServ\www\train2\protected\controllers\SiteController.php(98): CController->render('login', Array)
#14 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#15 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#16 C:\AppServ\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#17 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('login')
#19 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#20 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#21 C:\AppServ\www\train2\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/train2/index.php/site/login
HTTP_REFERER=http://localhost/train2/index.php/site/login
---
2014/10/21 19:09:24 [error] [exception.CException] exception 'CException' with message 'Property "CWebUser.username" is not defined.' in C:\AppServ\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\AppServ\www\yii\framework\web\auth\CWebUser.php(140): CComponent->__get('username')
#1 C:\AppServ\www\train2\protected\views\layouts\main.php(38): CWebUser->__get('username')
#2 C:\AppServ\www\yii\framework\web\CBaseController.php(126): require('C:\AppServ\www\...')
#3 C:\AppServ\www\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\AppServ\www\...', Array, true)
#4 C:\AppServ\www\yii\framework\web\widgets\CContentDecorator.php(76): CBaseController->renderFile('C:\AppServ\www\...', Array, true)
#5 C:\AppServ\www\yii\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate('<div id="conten...')
#6 C:\AppServ\www\yii\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput('<div id="conten...')
#7 C:\AppServ\www\yii\framework\web\CBaseController.php(206): COutputProcessor->run()
#8 C:\AppServ\www\yii\framework\web\CBaseController.php(300): CBaseController->endWidget('CContentDecorat...')
#9 C:\AppServ\www\train2\protected\views\layouts\column1.php(6): CBaseController->endContent()
#10 C:\AppServ\www\yii\framework\web\CBaseController.php(126): require('C:\AppServ\www\...')
#11 C:\AppServ\www\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\AppServ\www\...', Array, true)
#12 C:\AppServ\www\yii\framework\web\CController.php(784): CBaseController->renderFile('C:\AppServ\www\...', Array, true)
#13 C:\AppServ\www\train2\protected\controllers\SiteController.php(32): CController->render('index')
#14 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#15 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#16 C:\AppServ\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#17 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('')
#19 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#20 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#21 C:\AppServ\www\train2\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/train2/index.php
HTTP_REFERER=http://localhost/train2/index.php/site/login
---
2014/10/21 19:09:25 [error] [exception.CException] exception 'CException' with message 'Property "CWebUser.username" is not defined.' in C:\AppServ\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\AppServ\www\yii\framework\web\auth\CWebUser.php(140): CComponent->__get('username')
#1 C:\AppServ\www\train2\protected\views\layouts\main.php(38): CWebUser->__get('username')
#2 C:\AppServ\www\yii\framework\web\CBaseController.php(126): require('C:\AppServ\www\...')
#3 C:\AppServ\www\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\AppServ\www\...', Array, true)
#4 C:\AppServ\www\yii\framework\web\widgets\CContentDecorator.php(76): CBaseController->renderFile('C:\AppServ\www\...', Array, true)
#5 C:\AppServ\www\yii\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate('<div id="conten...')
#6 C:\AppServ\www\yii\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput('<div id="conten...')
#7 C:\AppServ\www\yii\framework\web\CBaseController.php(206): COutputProcessor->run()
#8 C:\AppServ\www\yii\framework\web\CBaseController.php(300): CBaseController->endWidget('CContentDecorat...')
#9 C:\AppServ\www\train2\protected\views\layouts\column1.php(6): CBaseController->endContent()
#10 C:\AppServ\www\yii\framework\web\CBaseController.php(126): require('C:\AppServ\www\...')
#11 C:\AppServ\www\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\AppServ\www\...', Array, true)
#12 C:\AppServ\www\yii\framework\web\CController.php(784): CBaseController->renderFile('C:\AppServ\www\...', Array, true)
#13 C:\AppServ\www\train2\protected\controllers\SiteController.php(98): CController->render('login', Array)
#14 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#15 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#16 C:\AppServ\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#17 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('login')
#19 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#20 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#21 C:\AppServ\www\train2\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/train2/index.php/site/login
HTTP_REFERER=http://localhost/train2/
---
2014/10/21 19:09:39 [error] [exception.CException] exception 'CException' with message 'Property "CWebUser._id" is not defined.' in C:\AppServ\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\AppServ\www\yii\framework\web\auth\CWebUser.php(140): CComponent->__get('_id')
#1 C:\AppServ\www\train2\protected\views\layouts\main.php(38): CWebUser->__get('_id')
#2 C:\AppServ\www\yii\framework\web\CBaseController.php(126): require('C:\AppServ\www\...')
#3 C:\AppServ\www\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\AppServ\www\...', Array, true)
#4 C:\AppServ\www\yii\framework\web\widgets\CContentDecorator.php(76): CBaseController->renderFile('C:\AppServ\www\...', Array, true)
#5 C:\AppServ\www\yii\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate('<div id="conten...')
#6 C:\AppServ\www\yii\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput('<div id="conten...')
#7 C:\AppServ\www\yii\framework\web\CBaseController.php(206): COutputProcessor->run()
#8 C:\AppServ\www\yii\framework\web\CBaseController.php(300): CBaseController->endWidget('CContentDecorat...')
#9 C:\AppServ\www\train2\protected\views\layouts\column1.php(6): CBaseController->endContent()
#10 C:\AppServ\www\yii\framework\web\CBaseController.php(126): require('C:\AppServ\www\...')
#11 C:\AppServ\www\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\AppServ\www\...', Array, true)
#12 C:\AppServ\www\yii\framework\web\CController.php(784): CBaseController->renderFile('C:\AppServ\www\...', Array, true)
#13 C:\AppServ\www\train2\protected\controllers\SiteController.php(98): CController->render('login', Array)
#14 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#15 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#16 C:\AppServ\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#17 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('login')
#19 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#20 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#21 C:\AppServ\www\train2\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/train2/index.php/site/login
---
2014/10/21 19:10:08 [error] [exception.CException] exception 'CException' with message 'Property "CWebUser._id" is not defined.' in C:\AppServ\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\AppServ\www\yii\framework\web\auth\CWebUser.php(140): CComponent->__get('_id')
#1 C:\AppServ\www\train2\protected\views\layouts\main.php(38): CWebUser->__get('_id')
#2 C:\AppServ\www\yii\framework\web\CBaseController.php(126): require('C:\AppServ\www\...')
#3 C:\AppServ\www\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\AppServ\www\...', Array, true)
#4 C:\AppServ\www\yii\framework\web\widgets\CContentDecorator.php(76): CBaseController->renderFile('C:\AppServ\www\...', Array, true)
#5 C:\AppServ\www\yii\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate('<div id="conten...')
#6 C:\AppServ\www\yii\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput('<div id="conten...')
#7 C:\AppServ\www\yii\framework\web\CBaseController.php(206): COutputProcessor->run()
#8 C:\AppServ\www\yii\framework\web\CBaseController.php(300): CBaseController->endWidget('CContentDecorat...')
#9 C:\AppServ\www\train2\protected\views\layouts\column1.php(6): CBaseController->endContent()
#10 C:\AppServ\www\yii\framework\web\CBaseController.php(126): require('C:\AppServ\www\...')
#11 C:\AppServ\www\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\AppServ\www\...', Array, true)
#12 C:\AppServ\www\yii\framework\web\CController.php(784): CBaseController->renderFile('C:\AppServ\www\...', Array, true)
#13 C:\AppServ\www\train2\protected\controllers\SiteController.php(32): CController->render('index')
#14 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#15 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#16 C:\AppServ\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#17 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('')
#19 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#20 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#21 C:\AppServ\www\train2\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/train2/
---
2014/10/21 19:19:12 [error] [php] YiiBase::include(User.php) [<a href='yiibase.include'>yiibase.include</a>]: failed to open stream: No such file or directory (C:\AppServ\www\yii\framework\YiiBase.php:421)
Stack trace:
#0 C:\AppServ\www\train2\protected\controllers\UserloginController.php(136): spl_autoload_call()
#1 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): UserloginController->actionAdmin()
#2 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#3 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(133): UserloginController->runAction()
#4 C:\AppServ\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#5 C:\AppServ\www\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#6 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): UserloginController->filterAccessControl()
#7 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#8 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#9 C:\AppServ\www\yii\framework\web\CController.php(265): UserloginController->runActionWithFilters()
#10 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): UserloginController->run()
#11 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#13 C:\AppServ\www\train2\index.php(13): CWebApplication->run()
REQUEST_URI=/train2/index.php/userlogin/admin
in C:\AppServ\www\train2\protected\controllers\UserloginController.php (136)
in C:\AppServ\www\train2\index.php (13)
2014/10/21 19:19:33 [error] [php] YiiBase::include(User.php) [<a href='yiibase.include'>yiibase.include</a>]: failed to open stream: No such file or directory (C:\AppServ\www\yii\framework\YiiBase.php:421)
Stack trace:
#0 C:\AppServ\www\train2\protected\controllers\UserloginController.php(136): spl_autoload_call()
#1 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): UserloginController->actionAdmin()
#2 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#3 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(133): UserloginController->runAction()
#4 C:\AppServ\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#5 C:\AppServ\www\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#6 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): UserloginController->filterAccessControl()
#7 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#8 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#9 C:\AppServ\www\yii\framework\web\CController.php(265): UserloginController->runActionWithFilters()
#10 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): UserloginController->run()
#11 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#13 C:\AppServ\www\train2\index.php(13): CWebApplication->run()
REQUEST_URI=/train2/index.php/userlogin/admin
in C:\AppServ\www\train2\protected\controllers\UserloginController.php (136)
in C:\AppServ\www\train2\index.php (13)
2014/10/21 19:20:33 [error] [php] YiiBase::include(User.php) [<a href='yiibase.include'>yiibase.include</a>]: failed to open stream: No such file or directory (C:\AppServ\www\yii\framework\YiiBase.php:421)
Stack trace:
#0 C:\AppServ\www\train2\protected\controllers\UserloginController.php(136): spl_autoload_call()
#1 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): UserloginController->actionAdmin()
#2 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#3 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(133): UserloginController->runAction()
#4 C:\AppServ\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#5 C:\AppServ\www\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#6 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): UserloginController->filterAccessControl()
#7 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#8 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#9 C:\AppServ\www\yii\framework\web\CController.php(265): UserloginController->runActionWithFilters()
#10 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): UserloginController->run()
#11 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#13 C:\AppServ\www\train2\index.php(13): CWebApplication->run()
REQUEST_URI=/train2/index.php/userlogin/admin
in C:\AppServ\www\train2\protected\controllers\UserloginController.php (136)
in C:\AppServ\www\train2\index.php (13)
2014/10/21 19:23:43 [error] [php] YiiBase::include(User.php) [<a href='yiibase.include'>yiibase.include</a>]: failed to open stream: No such file or directory (C:\AppServ\www\yii\framework\YiiBase.php:421)
Stack trace:
#0 C:\AppServ\www\train2\protected\controllers\UserloginController.php(136): spl_autoload_call()
#1 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): UserloginController->actionAdmin()
#2 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#3 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(133): UserloginController->runAction()
#4 C:\AppServ\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#5 C:\AppServ\www\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#6 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): UserloginController->filterAccessControl()
#7 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#8 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#9 C:\AppServ\www\yii\framework\web\CController.php(265): UserloginController->runActionWithFilters()
#10 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): UserloginController->run()
#11 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#13 C:\AppServ\www\train2\index.php(13): CWebApplication->run()
REQUEST_URI=/train2/index.php/userlogin/admin
in C:\AppServ\www\train2\protected\controllers\UserloginController.php (136)
in C:\AppServ\www\train2\index.php (13)
2014/10/21 19:24:24 [error] [php] YiiBase::include(User.php) [<a href='yiibase.include'>yiibase.include</a>]: failed to open stream: No such file or directory (C:\AppServ\www\yii\framework\YiiBase.php:421)
Stack trace:
#0 C:\AppServ\www\train2\protected\controllers\UserloginController.php(136): spl_autoload_call()
#1 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): UserloginController->actionAdmin()
#2 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#3 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(133): UserloginController->runAction()
#4 C:\AppServ\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#5 C:\AppServ\www\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#6 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): UserloginController->filterAccessControl()
#7 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#8 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#9 C:\AppServ\www\yii\framework\web\CController.php(265): UserloginController->runActionWithFilters()
#10 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): UserloginController->run()
#11 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#13 C:\AppServ\www\train2\index.php(13): CWebApplication->run()
REQUEST_URI=/train2/index.php/userlogin/admin
in C:\AppServ\www\train2\protected\controllers\UserloginController.php (136)
in C:\AppServ\www\train2\index.php (13)
2014/10/21 19:27:57 [error] [php] YiiBase::include(User.php) [<a href='yiibase.include'>yiibase.include</a>]: failed to open stream: No such file or directory (C:\AppServ\www\yii\framework\YiiBase.php:421)
Stack trace:
#0 C:\AppServ\www\train2\protected\controllers\UserloginController.php(136): spl_autoload_call()
#1 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): UserloginController->actionAdmin()
#2 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#3 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(133): UserloginController->runAction()
#4 C:\AppServ\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#5 C:\AppServ\www\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#6 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): UserloginController->filterAccessControl()
#7 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#8 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#9 C:\AppServ\www\yii\framework\web\CController.php(265): UserloginController->runActionWithFilters()
#10 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): UserloginController->run()
#11 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#13 C:\AppServ\www\train2\index.php(13): CWebApplication->run()
REQUEST_URI=/train2/index.php/userlogin/admin
in C:\AppServ\www\train2\protected\controllers\UserloginController.php (136)
in C:\AppServ\www\train2\index.php (13)
2014/10/21 19:30:07 [error] [php] YiiBase::include(User.php) [<a href='yiibase.include'>yiibase.include</a>]: failed to open stream: No such file or directory (C:\AppServ\www\yii\framework\YiiBase.php:421)
Stack trace:
#0 C:\AppServ\www\train2\protected\controllers\UserloginController.php(136): spl_autoload_call()
#1 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): UserloginController->actionAdmin()
#2 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#3 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(133): UserloginController->runAction()
#4 C:\AppServ\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#5 C:\AppServ\www\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#6 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): UserloginController->filterAccessControl()
#7 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#8 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#9 C:\AppServ\www\yii\framework\web\CController.php(265): UserloginController->runActionWithFilters()
#10 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): UserloginController->run()
#11 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#13 C:\AppServ\www\train2\index.php(13): CWebApplication->run()
REQUEST_URI=/train2/index.php/userlogin/admin
in C:\AppServ\www\train2\protected\controllers\UserloginController.php (136)
in C:\AppServ\www\train2\index.php (13)
2014/10/21 19:33:57 [error] [php] YiiBase::include(User.php) [<a href='yiibase.include'>yiibase.include</a>]: failed to open stream: No such file or directory (C:\AppServ\www\yii\framework\YiiBase.php:421)
Stack trace:
#0 C:\AppServ\www\train2\protected\controllers\UserloginController.php(136): spl_autoload_call()
#1 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): UserloginController->actionAdmin()
#2 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#3 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(133): UserloginController->runAction()
#4 C:\AppServ\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#5 C:\AppServ\www\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#6 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): UserloginController->filterAccessControl()
#7 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#8 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#9 C:\AppServ\www\yii\framework\web\CController.php(265): UserloginController->runActionWithFilters()
#10 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): UserloginController->run()
#11 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#13 C:\AppServ\www\train2\index.php(13): CWebApplication->run()
REQUEST_URI=/train2/index.php/userlogin/admin
in C:\AppServ\www\train2\protected\controllers\UserloginController.php (136)
in C:\AppServ\www\train2\index.php (13)
2014/10/21 19:34:19 [error] [php] YiiBase::include(User.php) [<a href='yiibase.include'>yiibase.include</a>]: failed to open stream: No such file or directory (C:\AppServ\www\yii\framework\YiiBase.php:421)
Stack trace:
#0 C:\AppServ\www\train2\protected\controllers\UserloginController.php(136): spl_autoload_call()
#1 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): UserloginController->actionAdmin()
#2 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#3 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(133): UserloginController->runAction()
#4 C:\AppServ\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#5 C:\AppServ\www\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#6 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): UserloginController->filterAccessControl()
#7 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#8 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#9 C:\AppServ\www\yii\framework\web\CController.php(265): UserloginController->runActionWithFilters()
#10 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): UserloginController->run()
#11 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#13 C:\AppServ\www\train2\index.php(13): CWebApplication->run()
REQUEST_URI=/train2/index.php/userlogin/admin
in C:\AppServ\www\train2\protected\controllers\UserloginController.php (136)
in C:\AppServ\www\train2\index.php (13)
2014/10/21 19:35:46 [error] [php] YiiBase::include(User.php) [<a href='yiibase.include'>yiibase.include</a>]: failed to open stream: No such file or directory (C:\AppServ\www\yii\framework\YiiBase.php:421)
Stack trace:
#0 C:\AppServ\www\train2\protected\controllers\UserloginController.php(136): spl_autoload_call()
#1 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): UserloginController->actionAdmin()
#2 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#3 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(133): UserloginController->runAction()
#4 C:\AppServ\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#5 C:\AppServ\www\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#6 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): UserloginController->filterAccessControl()
#7 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#8 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#9 C:\AppServ\www\yii\framework\web\CController.php(265): UserloginController->runActionWithFilters()
#10 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): UserloginController->run()
#11 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#13 C:\AppServ\www\train2\index.php(13): CWebApplication->run()
REQUEST_URI=/train2/index.php/userlogin/admin
in C:\AppServ\www\train2\protected\controllers\UserloginController.php (136)
in C:\AppServ\www\train2\index.php (13)
2014/10/21 19:42:21 [error] [php] YiiBase::include(User.php) [<a href='yiibase.include'>yiibase.include</a>]: failed to open stream: No such file or directory (C:\AppServ\www\yii\framework\YiiBase.php:421)
Stack trace:
#0 C:\AppServ\www\yii\framework\db\ar\CActiveRecord.php(386): spl_autoload_call()
#1 C:\AppServ\www\yii\framework\web\CActiveDataProvider.php(74): model()
#2 C:\AppServ\www\train2\protected\controllers\UserloginController.php(125): CActiveDataProvider->__construct()
#3 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): UserloginController->actionIndex()
#4 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(133): UserloginController->runAction()
#6 C:\AppServ\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\AppServ\www\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): UserloginController->filterAccessControl()
#9 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\AppServ\www\yii\framework\web\CController.php(265): UserloginController->runActionWithFilters()
#12 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): UserloginController->run()
#13 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\AppServ\www\train2\index.php(13): CWebApplication->run()
REQUEST_URI=/train2/index.php/userlogin/index
in C:\AppServ\www\train2\protected\controllers\UserloginController.php (125)
in C:\AppServ\www\train2\index.php (13)
2014/10/21 19:47:02 [error] [exception.CException] exception 'CException' with message 'Property "CWebUser.homeUrl" is not defined.' in C:\AppServ\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\AppServ\www\yii\framework\web\auth\CWebUser.php(140): CComponent->__get('homeUrl')
#1 C:\AppServ\www\train2\protected\controllers\SiteController.php(95): CWebUser->__get('homeUrl')
#2 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#3 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\AppServ\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('login')
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#8 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 C:\AppServ\www\train2\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/train2/index.php/site/login
HTTP_REFERER=http://localhost/train2/index.php/site/login
---
2014/10/21 19:48:50 [error] [php] YiiBase::include(User.php) [<a href='yiibase.include'>yiibase.include</a>]: failed to open stream: No such file or directory (C:\AppServ\www\yii\framework\YiiBase.php:421)
Stack trace:
#0 C:\AppServ\www\yii\framework\db\ar\CActiveRecord.php(386): spl_autoload_call()
#1 C:\AppServ\www\yii\framework\web\CActiveDataProvider.php(74): model()
#2 C:\AppServ\www\train2\protected\controllers\UserloginController.php(125): CActiveDataProvider->__construct()
#3 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): UserloginController->actionIndex()
#4 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(133): UserloginController->runAction()
#6 C:\AppServ\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\AppServ\www\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): UserloginController->filterAccessControl()
#9 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\AppServ\www\yii\framework\web\CController.php(265): UserloginController->runActionWithFilters()
#12 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): UserloginController->run()
#13 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\AppServ\www\train2\index.php(13): CWebApplication->run()
REQUEST_URI=/train2/index.php/userlogin/index
in C:\AppServ\www\train2\protected\controllers\UserloginController.php (125)
in C:\AppServ\www\train2\index.php (13)
2014/10/21 19:54:33 [error] [exception.CException] exception 'CException' with message 'Property "CWebApplication.returnUrl" is not defined.' in C:\AppServ\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\AppServ\www\yii\framework\base\CModule.php(105): CComponent->__get('returnUrl')
#1 C:\AppServ\www\train2\protected\controllers\SiteController.php(107): CModule->__get('returnUrl')
#2 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogout()
#3 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\AppServ\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('logout')
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/logout')
#8 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 C:\AppServ\www\train2\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/train2/index.php/site/logout
HTTP_REFERER=http://localhost/train2/index.php/site/login
---
2014/10/21 19:54:46 [error] [exception.CException] exception 'CException' with message 'Property "CWebApplication.returnUrl" is not defined.' in C:\AppServ\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\AppServ\www\yii\framework\base\CModule.php(105): CComponent->__get('returnUrl')
#1 C:\AppServ\www\train2\protected\controllers\SiteController.php(107): CModule->__get('returnUrl')
#2 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogout()
#3 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\AppServ\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('logout')
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/logout')
#8 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 C:\AppServ\www\train2\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/train2/index.php/site/logout
HTTP_REFERER=http://localhost/train2/index.php/site/login
---
2014/10/21 20:05:08 [error] [exception.CException] exception 'CException' with message 'Property "CWebApplication.returnUrl" is not defined.' in C:\AppServ\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\AppServ\www\yii\framework\base\CModule.php(105): CComponent->__get('returnUrl')
#1 C:\AppServ\www\train2\protected\controllers\SiteController.php(107): CModule->__get('returnUrl')
#2 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogout()
#3 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\AppServ\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('logout')
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/logout')
#8 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 C:\AppServ\www\train2\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/train2/index.php/site/logout
HTTP_REFERER=http://localhost/train2/index.php
---
2014/10/21 22:32:55 [error] [php] YiiBase::include(User.php) [<a href='yiibase.include'>yiibase.include</a>]: failed to open stream: No such file or directory (C:\AppServ\www\yii\framework\YiiBase.php:421)
Stack trace:
#0 C:\AppServ\www\yii\framework\db\ar\CActiveRecord.php(386): spl_autoload_call()
#1 C:\AppServ\www\yii\framework\web\CActiveDataProvider.php(74): model()
#2 C:\AppServ\www\train2\protected\controllers\UserloginController.php(125): CActiveDataProvider->__construct()
#3 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): UserloginController->actionIndex()
#4 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(133): UserloginController->runAction()
#6 C:\AppServ\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\AppServ\www\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): UserloginController->filterAccessControl()
#9 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\AppServ\www\yii\framework\web\CController.php(265): UserloginController->runActionWithFilters()
#12 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): UserloginController->run()
#13 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\AppServ\www\train2\index.php(13): CWebApplication->run()
REQUEST_URI=/train2/index.php/userlogin/index
in C:\AppServ\www\train2\protected\controllers\UserloginController.php (125)
in C:\AppServ\www\train2\index.php (13)
2014/10/21 22:36:58 [error] [php] YiiBase::include(User.php) [<a href='yiibase.include'>yiibase.include</a>]: failed to open stream: No such file or directory (C:\AppServ\www\yii\framework\YiiBase.php:421)
Stack trace:
#0 C:\AppServ\www\yii\framework\db\ar\CActiveRecord.php(386): spl_autoload_call()
#1 C:\AppServ\www\yii\framework\web\CActiveDataProvider.php(74): model()
#2 C:\AppServ\www\train2\protected\controllers\UserloginController.php(125): CActiveDataProvider->__construct()
#3 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): UserloginController->actionIndex()
#4 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(133): UserloginController->runAction()
#6 C:\AppServ\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\AppServ\www\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): UserloginController->filterAccessControl()
#9 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\AppServ\www\yii\framework\web\CController.php(265): UserloginController->runActionWithFilters()
#12 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): UserloginController->run()
#13 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\AppServ\www\train2\index.php(13): CWebApplication->run()
REQUEST_URI=/train2/index.php/userlogin/index
in C:\AppServ\www\train2\protected\controllers\UserloginController.php (125)
in C:\AppServ\www\train2\index.php (13)
2014/10/21 22:37:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "เร".' in C:\AppServ\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('??????')
#1 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\AppServ\www\train2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/train2/index.php/%E0%B9%80%E0%B8%A3
---
2014/10/21 22:37:32 [error] [php] YiiBase::include(User.php) [<a href='yiibase.include'>yiibase.include</a>]: failed to open stream: No such file or directory (C:\AppServ\www\yii\framework\YiiBase.php:421)
Stack trace:
#0 C:\AppServ\www\yii\framework\db\ar\CActiveRecord.php(386): spl_autoload_call()
#1 C:\AppServ\www\yii\framework\web\CActiveDataProvider.php(74): model()
#2 C:\AppServ\www\train2\protected\controllers\UserloginController.php(125): CActiveDataProvider->__construct()
#3 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): UserloginController->actionIndex()
#4 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(133): UserloginController->runAction()
#6 C:\AppServ\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\AppServ\www\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): UserloginController->filterAccessControl()
#9 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\AppServ\www\yii\framework\web\CController.php(265): UserloginController->runActionWithFilters()
#12 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): UserloginController->run()
#13 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\AppServ\www\train2\index.php(13): CWebApplication->run()
REQUEST_URI=/train2/index.php/userlogin
in C:\AppServ\www\train2\protected\controllers\UserloginController.php (125)
in C:\AppServ\www\train2\index.php (13)
2014/10/21 22:39:58 [error] [php] YiiBase::include(User.php) [<a href='yiibase.include'>yiibase.include</a>]: failed to open stream: No such file or directory (C:\AppServ\www\yii\framework\YiiBase.php:421)
Stack trace:
#0 C:\AppServ\www\yii\framework\db\ar\CActiveRecord.php(386): spl_autoload_call()
#1 C:\AppServ\www\yii\framework\web\CActiveDataProvider.php(74): model()
#2 C:\AppServ\www\train2\protected\controllers\UserloginController.php(125): CActiveDataProvider->__construct()
#3 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): UserloginController->actionIndex()
#4 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(133): UserloginController->runAction()
#6 C:\AppServ\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\AppServ\www\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): UserloginController->filterAccessControl()
#9 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\AppServ\www\yii\framework\web\CController.php(265): UserloginController->runActionWithFilters()
#12 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): UserloginController->run()
#13 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\AppServ\www\train2\index.php(13): CWebApplication->run()
REQUEST_URI=/train2/index.php/userlogin/index
in C:\AppServ\www\train2\protected\controllers\UserloginController.php (125)
in C:\AppServ\www\train2\index.php (13)
2014/10/21 22:40:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "userlogin/index".' in C:\AppServ\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/index')
#1 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\AppServ\www\train2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/train2/index.php/userlogin/index
HTTP_REFERER=http://localhost/train2/index.php/
---
2014/10/21 22:41:13 [error] [exception.CException] exception 'CException' with message 'Userlogin and its behaviors do not have a method or closure named "validatePassword".' in C:\AppServ\www\yii\framework\base\CComponent.php:265
Stack trace:
#0 C:\AppServ\www\yii\framework\db\ar\CActiveRecord.php(225): CComponent->__call('validatePasswor...', Array)
#1 [internal function]: CActiveRecord->__call('validatePasswor...', Array)
#2 C:\AppServ\www\train2\protected\components\UserIdentity.php(28): Userlogin->validatePassword('admin')
#3 C:\AppServ\www\train2\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#4 C:\AppServ\www\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#5 C:\AppServ\www\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#6 C:\AppServ\www\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#7 C:\AppServ\www\train2\protected\controllers\SiteController.php(94): CModel->validate()
#8 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#9 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\AppServ\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('login')
#13 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#14 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\AppServ\www\train2\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/train2/index.php/site/login
HTTP_REFERER=http://localhost/train2/index.php/site/login
---
2014/10/21 22:42:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "g".' in C:\AppServ\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('g')
#1 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\AppServ\www\train2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/train2/index.php/g
---
2014/10/21 22:56:11 [error] [exception.CException] exception 'CException' with message 'Userlogin and its behaviors do not have a method or closure named "validatePassword".' in C:\AppServ\www\yii\framework\base\CComponent.php:265
Stack trace:
#0 C:\AppServ\www\yii\framework\db\ar\CActiveRecord.php(225): CComponent->__call('validatePasswor...', Array)
#1 [internal function]: CActiveRecord->__call('validatePasswor...', Array)
#2 C:\AppServ\www\train2\protected\components\UserIdentity.php(28): Userlogin->validatePassword('123')
#3 C:\AppServ\www\train2\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#4 C:\AppServ\www\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#5 C:\AppServ\www\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#6 C:\AppServ\www\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#7 C:\AppServ\www\train2\protected\controllers\SiteController.php(94): CModel->validate()
#8 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#9 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\AppServ\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('login')
#13 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#14 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\AppServ\www\train2\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/train2/index.php/site/login
HTTP_REFERER=http://localhost/train2/index.php/site/login
---
2014/10/21 23:14:17 [error] [exception.CException] exception 'CException' with message 'Userlogin and its behaviors do not have a method or closure named "validatePassword".' in C:\AppServ\www\yii\framework\base\CComponent.php:265
Stack trace:
#0 C:\AppServ\www\yii\framework\db\ar\CActiveRecord.php(225): CComponent->__call('validatePasswor...', Array)
#1 [internal function]: CActiveRecord->__call('validatePasswor...', Array)
#2 C:\AppServ\www\train2\protected\components\UserIdentity.php(25): Userlogin->validatePassword('12345')
#3 C:\AppServ\www\train2\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#4 C:\AppServ\www\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#5 C:\AppServ\www\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#6 C:\AppServ\www\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#7 C:\AppServ\www\train2\protected\controllers\SiteController.php(94): CModel->validate()
#8 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#9 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\AppServ\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('login')
#13 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#14 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\AppServ\www\train2\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/train2/index.php/site/login
HTTP_REFERER=http://localhost/train2/index.php/site/login
---
2014/10/21 23:15:57 [error] [exception.CException] exception 'CException' with message 'Userlogin and its behaviors do not have a method or closure named "validatePassword".' in C:\AppServ\www\yii\framework\base\CComponent.php:265
Stack trace:
#0 C:\AppServ\www\yii\framework\db\ar\CActiveRecord.php(225): CComponent->__call('validatePasswor...', Array)
#1 [internal function]: CActiveRecord->__call('validatePasswor...', Array)
#2 C:\AppServ\www\train2\protected\components\UserIdentity.php(25): Userlogin->validatePassword('12345')
#3 C:\AppServ\www\train2\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#4 C:\AppServ\www\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#5 C:\AppServ\www\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#6 C:\AppServ\www\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#7 C:\AppServ\www\train2\protected\controllers\SiteController.php(94): CModel->validate()
#8 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#9 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\AppServ\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('login')
#13 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#14 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\AppServ\www\train2\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/train2/index.php/site/login
HTTP_REFERER=http://localhost/train2/index.php/site/login
---
2014/10/21 23:16:02 [error] [exception.CException] exception 'CException' with message 'Userlogin and its behaviors do not have a method or closure named "validatePassword".' in C:\AppServ\www\yii\framework\base\CComponent.php:265
Stack trace:
#0 C:\AppServ\www\yii\framework\db\ar\CActiveRecord.php(225): CComponent->__call('validatePasswor...', Array)
#1 [internal function]: CActiveRecord->__call('validatePasswor...', Array)
#2 C:\AppServ\www\train2\protected\components\UserIdentity.php(25): Userlogin->validatePassword('12345')
#3 C:\AppServ\www\train2\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#4 C:\AppServ\www\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#5 C:\AppServ\www\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#6 C:\AppServ\www\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#7 C:\AppServ\www\train2\protected\controllers\SiteController.php(94): CModel->validate()
#8 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#9 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\AppServ\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('login')
#13 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#14 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\AppServ\www\train2\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/train2/index.php/site/login
HTTP_REFERER=http://localhost/train2/index.php/site/login
---
2014/10/21 23:16:46 [error] [exception.CException] exception 'CException' with message 'Property "Userlogin.id_user" is not defined.' in C:\AppServ\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\AppServ\www\yii\framework\db\ar\CActiveRecord.php(143): CComponent->__get('id_user')
#1 C:\AppServ\www\train2\protected\components\UserIdentity.php(28): CActiveRecord->__get('id_user')
#2 C:\AppServ\www\train2\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#3 C:\AppServ\www\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#4 C:\AppServ\www\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\AppServ\www\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#6 C:\AppServ\www\train2\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\AppServ\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#14 C:\AppServ\www\train2\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/train2/index.php/site/login
HTTP_REFERER=http://localhost/train2/index.php/site/login
---
2014/10/21 23:17:28 [error] [exception.CException] exception 'CException' with message 'Property "Userlogin.id_user" is not defined.' in C:\AppServ\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\AppServ\www\yii\framework\db\ar\CActiveRecord.php(143): CComponent->__get('id_user')
#1 C:\AppServ\www\train2\protected\components\UserIdentity.php(27): CActiveRecord->__get('id_user')
#2 C:\AppServ\www\train2\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#3 C:\AppServ\www\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#4 C:\AppServ\www\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\AppServ\www\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#6 C:\AppServ\www\train2\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\AppServ\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#14 C:\AppServ\www\train2\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/train2/index.php/site/login
HTTP_REFERER=http://localhost/train2/index.php/site/login
---
2014/10/21 23:18:09 [error] [exception.CException] exception 'CException' with message 'Property "UserIdentity._id" is not defined.' in C:\AppServ\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\AppServ\www\train2\protected\components\UserIdentity.php(37): CComponent->__get('_id')
#1 C:\AppServ\www\yii\framework\web\auth\CWebUser.php(223): UserIdentity->getId()
#2 C:\AppServ\www\train2\protected\models\LoginForm.php(71): CWebUser->login(Object(UserIdentity), 0)
#3 C:\AppServ\www\train2\protected\controllers\SiteController.php(94): LoginForm->login()
#4 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#5 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 C:\AppServ\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('login')
#9 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#10 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 C:\AppServ\www\train2\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/train2/index.php/site/login
HTTP_REFERER=http://localhost/train2/index.php/site/login
---
2014/10/21 23:18:29 [error] [exception.CException] exception 'CException' with message 'Property "UserIdentity.id" is read only.' in C:\AppServ\www\yii\framework\base\CComponent.php:170
Stack trace:
#0 C:\AppServ\www\train2\protected\components\UserIdentity.php(27): CComponent->__set('id', '8')
#1 C:\AppServ\www\train2\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#2 C:\AppServ\www\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#3 C:\AppServ\www\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#4 C:\AppServ\www\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#5 C:\AppServ\www\train2\protected\controllers\SiteController.php(94): CModel->validate()
#6 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\AppServ\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('login')
#11 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#12 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#13 C:\AppServ\www\train2\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/train2/index.php/site/login
HTTP_REFERER=http://localhost/train2/index.php/site/login
---
2014/10/21 23:18:57 [error] [exception.CException] exception 'CException' with message 'Property "Userlogin._id" is not defined.' in C:\AppServ\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\AppServ\www\yii\framework\db\ar\CActiveRecord.php(143): CComponent->__get('_id')
#1 C:\AppServ\www\train2\protected\components\UserIdentity.php(27): CActiveRecord->__get('_id')
#2 C:\AppServ\www\train2\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#3 C:\AppServ\www\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#4 C:\AppServ\www\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\AppServ\www\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#6 C:\AppServ\www\train2\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\AppServ\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#14 C:\AppServ\www\train2\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/train2/index.php/site/login
HTTP_REFERER=http://localhost/train2/index.php/site/login
---
2014/10/21 23:19:24 [error] [exception.CException] exception 'CException' with message 'Property "CWebApplication.returnUrl" is not defined.' in C:\AppServ\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\AppServ\www\yii\framework\base\CModule.php(105): CComponent->__get('returnUrl')
#1 C:\AppServ\www\train2\protected\controllers\SiteController.php(107): CModule->__get('returnUrl')
#2 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogout()
#3 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\AppServ\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('logout')
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/logout')
#8 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 C:\AppServ\www\train2\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/train2/index.php/site/logout
HTTP_REFERER=http://localhost/train2/index.php/userlogin/index
---
2014/10/21 23:19:44 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\AppServ\www\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\AppServ\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/admin')
#9 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\AppServ\www\train2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/train2/index.php/userlogin/admin
HTTP_REFERER=http://localhost/train2/index.php/userlogin/create
---
2014/10/21 23:19:49 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\AppServ\www\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\AppServ\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/admin')
#9 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\AppServ\www\train2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/train2/index.php/userlogin/admin
HTTP_REFERER=http://localhost/train2/index.php/userlogin/create
---
2014/10/21 23:19:52 [error] [exception.CException] exception 'CException' with message 'Property "CWebApplication.returnUrl" is not defined.' in C:\AppServ\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\AppServ\www\yii\framework\base\CModule.php(105): CComponent->__get('returnUrl')
#1 C:\AppServ\www\train2\protected\controllers\SiteController.php(107): CModule->__get('returnUrl')
#2 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogout()
#3 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\AppServ\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('logout')
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/logout')
#8 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 C:\AppServ\www\train2\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/train2/index.php/site/logout
HTTP_REFERER=http://localhost/train2/index.php
---
2014/10/21 23:22:54 [error] [exception.CException] exception 'CException' with message 'Property "CWebApplication.returnUrl" is not defined.' in C:\AppServ\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\AppServ\www\yii\framework\base\CModule.php(105): CComponent->__get('returnUrl')
#1 C:\AppServ\www\train2\protected\controllers\SiteController.php(107): CModule->__get('returnUrl')
#2 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogout()
#3 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\AppServ\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('logout')
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/logout')
#8 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 C:\AppServ\www\train2\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/train2/index.php/site/logout
HTTP_REFERER=http://localhost/train2/index.php
---
2014/10/21 23:26:24 [error] [exception.CException] exception 'CException' with message 'Property "CWebApplication.returnUrl" is not defined.' in C:\AppServ\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\AppServ\www\yii\framework\base\CModule.php(105): CComponent->__get('returnUrl')
#1 C:\AppServ\www\train2\protected\controllers\SiteController.php(107): CModule->__get('returnUrl')
#2 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogout()
#3 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\AppServ\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('logout')
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/logout')
#8 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 C:\AppServ\www\train2\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/train2/index.php/site/logout
---
2014/10/21 23:26:37 [error] [exception.CException] exception 'CException' with message 'Property "CWebApplication.returnUrl" is not defined.' in C:\AppServ\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\AppServ\www\yii\framework\base\CModule.php(105): CComponent->__get('returnUrl')
#1 C:\AppServ\www\train2\protected\controllers\SiteController.php(107): CModule->__get('returnUrl')
#2 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogout()
#3 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\AppServ\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('logout')
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/logout')
#8 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 C:\AppServ\www\train2\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/train2/index.php/site/logout
HTTP_REFERER=http://localhost/train2/index.php
---
2014/10/21 23:28:30 [error] [exception.CException] exception 'CException' with message 'Property "CWebApplication.returnUrl" is not defined.' in C:\AppServ\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\AppServ\www\yii\framework\base\CModule.php(105): CComponent->__get('returnUrl')
#1 C:\AppServ\www\train2\protected\controllers\SiteController.php(108): CModule->__get('returnUrl')
#2 C:\AppServ\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogout()
#3 C:\AppServ\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\AppServ\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('logout')
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/logout')
#8 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 C:\AppServ\www\train2\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/train2/index.php/site/logout
HTTP_REFERER=http://localhost/train2/index.php
---
2014/10/21 23:31:59 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\AppServ\www\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\AppServ\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/admin')
#9 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\AppServ\www\train2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/train2/index.php/userlogin/admin
HTTP_REFERER=http://localhost/train2/index.php/site/login
---
2014/10/21 23:34:38 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\AppServ\www\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\AppServ\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/admin')
#9 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\AppServ\www\train2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/train2/index.php/userlogin/admin
HTTP_REFERER=http://localhost/train2/index.php/userlogin/create
---
2014/10/21 23:34:42 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\AppServ\www\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\AppServ\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/admin')
#9 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\AppServ\www\train2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/train2/index.php/userlogin/admin
HTTP_REFERER=http://localhost/train2/index.php/userlogin/create
---
2014/10/21 23:35:09 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\AppServ\www\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\AppServ\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/admin')
#9 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\AppServ\www\train2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/train2/index.php/userlogin/admin
HTTP_REFERER=http://localhost/train2/index.php/site/login
---
2014/10/21 23:36:31 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\AppServ\www\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\AppServ\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/admin')
#9 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\AppServ\www\train2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/train2/index.php/userlogin/admin
HTTP_REFERER=http://localhost/train2/index.php/userlogin/create
---
2014/10/21 23:36:39 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\AppServ\www\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\AppServ\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/admin')
#9 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\AppServ\www\train2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/train2/index.php/userlogin/admin
HTTP_REFERER=http://localhost/train2/index.php/userlogin/index
---
2014/10/21 23:39:12 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\AppServ\www\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\AppServ\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/admin')
#9 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\AppServ\www\train2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/train2/index.php/userlogin/admin
HTTP_REFERER=http://localhost/train2/index.php/userlogin/create
---
2014/10/21 23:41:30 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\AppServ\www\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\AppServ\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/admin')
#9 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\AppServ\www\train2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/train2/index.php/userlogin/admin
HTTP_REFERER=http://localhost/train2/index.php/userlogin/create
---
2014/10/21 23:41:34 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\AppServ\www\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\AppServ\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/admin')
#9 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\AppServ\www\train2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/train2/index.php/userlogin/admin
HTTP_REFERER=http://localhost/train2/index.php/userlogin/create
---
2014/10/21 23:41:38 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\AppServ\www\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\AppServ\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/admin')
#9 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\AppServ\www\train2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/train2/index.php/userlogin/admin
HTTP_REFERER=http://localhost/train2/index.php/userlogin/create
---
2014/10/21 23:42:22 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\AppServ\www\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\AppServ\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/admin')
#9 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\AppServ\www\train2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/train2/index.php/userlogin/admin
HTTP_REFERER=http://localhost/train2/index.php/userlogin/create
---
2014/10/21 23:42:41 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\AppServ\www\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\AppServ\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/admin')
#9 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\AppServ\www\train2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/train2/index.php/userlogin/admin
HTTP_REFERER=http://localhost/train2/index.php/userlogin/create
---
2014/10/21 23:46:11 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\AppServ\www\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\AppServ\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/admin')
#9 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\AppServ\www\train2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/train2/index.php/userlogin/admin
HTTP_REFERER=http://localhost/train2/index.php/userlogin/create
---
2014/10/21 23:46:14 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\AppServ\www\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\AppServ\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/admin')
#9 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\AppServ\www\train2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/train2/index.php/userlogin/admin
HTTP_REFERER=http://localhost/train2/index.php/userlogin/create
---
2014/10/21 23:48:42 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\AppServ\www\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\AppServ\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#8 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/index')
#9 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\AppServ\www\train2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/train2/index.php/userlogin/index
HTTP_REFERER=http://localhost/train2/index.php/userlogin/admin
---
2014/10/21 23:49:31 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\AppServ\www\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\AppServ\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#8 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/index')
#9 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\AppServ\www\train2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/train2/index.php/userlogin/index
HTTP_REFERER=http://localhost/train2/index.php/userlogin/create
---
2014/10/21 23:49:44 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\AppServ\www\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\AppServ\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#8 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/index')
#9 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\AppServ\www\train2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/train2/index.php/userlogin/index
HTTP_REFERER=http://localhost/train2/index.php/userlogin/admin
---
2014/10/21 23:49:51 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\AppServ\www\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\AppServ\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#8 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/index')
#9 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\AppServ\www\train2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/train2/index.php/userlogin/index
HTTP_REFERER=http://localhost/train2/index.php/userlogin/create
---
2014/10/21 23:50:00 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\AppServ\www\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\AppServ\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#8 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/index')
#9 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\AppServ\www\train2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/train2/index.php/userlogin/index
HTTP_REFERER=http://localhost/train2/index.php/site/login
---
2014/10/21 23:50:06 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\AppServ\www\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\AppServ\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#8 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/index')
#9 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\AppServ\www\train2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/train2/index.php/userlogin/index
HTTP_REFERER=http://localhost/train2/index.php/userlogin/create
---
2014/10/21 23:50:39 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\AppServ\www\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\AppServ\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#8 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/index')
#9 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\AppServ\www\train2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/train2/index.php/userlogin/index
HTTP_REFERER=http://localhost/train2/index.php/userlogin/create
---
2014/10/21 23:50:42 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\AppServ\www\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\AppServ\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#8 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/index')
#9 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\AppServ\www\train2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/train2/index.php/userlogin/index
HTTP_REFERER=http://localhost/train2/index.php/userlogin/create
---
2014/10/21 23:55:37 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\AppServ\www\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\AppServ\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('line/admin')
#9 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\AppServ\www\train2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/train2/index.php/line/admin
HTTP_REFERER=http://localhost/train2/index.php/line/index
---
2014/10/21 23:55:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "station/index".' in C:\AppServ\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('station/index')
#1 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\AppServ\www\train2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/train2/index.php/station/index
HTTP_REFERER=http://localhost/train2/index.php/line/index
---
2014/10/21 23:55:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "station/index".' in C:\AppServ\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('station/index')
#1 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\AppServ\www\train2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/train2/index.php/station/index
HTTP_REFERER=http://localhost/train2/index.php/station/index
---
2014/10/21 23:55:46 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "station/index".' in C:\AppServ\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('station/index')
#1 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\AppServ\www\train2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/train2/index.php/station/index
HTTP_REFERER=http://localhost/train2/index.php/station/index
---
2014/10/21 23:55:46 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "station/index".' in C:\AppServ\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('station/index')
#1 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\AppServ\www\train2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/train2/index.php/station/index
HTTP_REFERER=http://localhost/train2/index.php/station/index
---
2014/10/21 23:55:48 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "vehicle/index".' in C:\AppServ\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('vehicle/index')
#1 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\AppServ\www\train2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/train2/index.php/vehicle/index
HTTP_REFERER=http://localhost/train2/index.php/station/index
---
2014/10/21 23:55:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "driver/index".' in C:\AppServ\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('driver/index')
#1 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\AppServ\www\train2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/train2/index.php/driver/index
HTTP_REFERER=http://localhost/train2/index.php/vehicle/index
---
2014/10/21 23:56:24 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\AppServ\www\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\AppServ\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('delete')
#8 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/delet...')
#9 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\AppServ\www\train2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/train2/index.php/userlogin/delete/1
HTTP_REFERER=http://localhost/train2/index.php/userlogin/1
---
2014/10/21 23:56:38 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\AppServ\www\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\AppServ\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('delete')
#8 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/delet...')
#9 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\AppServ\www\train2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/train2/index.php/userlogin/delete/1?ajax=userlogin-grid
HTTP_REFERER=http://localhost/train2/index.php/userlogin/admin
---
2014/10/21 23:56:44 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\AppServ\www\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\AppServ\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('delete')
#8 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/delet...')
#9 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\AppServ\www\train2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/train2/index.php/userlogin/delete/1?ajax=userlogin-grid
HTTP_REFERER=http://localhost/train2/index.php/userlogin/admin
---
2014/10/21 23:56:49 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\AppServ\www\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\AppServ\www\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\AppServ\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('delete')
#8 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/delet...')
#9 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\AppServ\www\train2\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/train2/index.php/userlogin/delete/1?ajax=userlogin-grid
HTTP_REFERER=http://localhost/train2/index.php/userlogin/admin
---
2014/10/21 23:57:17 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\AppServ\www\yii\framework\web\CController.php:1118
Stack trace:
#0 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterPostOnly(Object(CFilterChain))
#1 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#2 C:\AppServ\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\AppServ\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\AppServ\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\AppServ\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\AppServ\www\yii\framework\web\CController.php(291): CFilterChain->run()
#7 C:\AppServ\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\AppServ\www\yii\framework\web\CWebApplication.php(282): CController->run('delete')
#9 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('userlogin/delet...')
#10 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 C:\AppServ\www\train2\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/train2/index.php/userlogin/delete/1
HTTP_REFERER=http://localhost/train2/index.php/userlogin/admin
---
2014/10/21 23:58:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "station/index".' in C:\AppServ\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\AppServ\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('station/index')
#1 C:\AppServ\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\AppServ\www\train2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/train2/index.php/station/index
HTTP_REFERER=http://localhost/train2/index.php/site/login
---
